src/modules/public/errorHandler/helpers/getMetaMaskErrorMessage.ts

```diff
@@ -0,0 +1,19 @@
+import { UnsupportedChainIdError } from '@web3-react/core';
+import {
+    NoEthereumProviderError,
+    UserRejectedRequestError as UserRejectedRequestErrorInjected,
+} from '@web3-react/injected-connector';
+
+export const getMetaMaskErrorMessage = (error: Error): string => {
+    if (error instanceof NoEthereumProviderError) {
+        return 'metamask.error.noExtension';
+    } else if (error instanceof UnsupportedChainIdError) {
+        return 'metamask.error.unsupportedNetwork';
+    } else if (error instanceof UserRejectedRequestErrorInjected) {
+        return 'metamask.error.unauthorized';
+    } else {
+        window.console.error(error);
+
```

Comment:
No need to use it as soon as message 'metamask.error.unknown' will be displayed in alert

Reply:
It's needed. 
'metamask.error.unknown' text -> [MetaMask] An unknown error occurred. Check the console for more details